YSI_Coding/y_inline/y_inline_extra.inc

```diff
@@ -72,63 +72,109 @@ Optional plugins:
 
 stock MySQL_PQueryInline(MySQL:handle, Func:cb<>, const query[], GLOBAL_TAG_TYPES:...)
 {
-	Indirect_Claim(cb);
-	mysql_format(handle, YSI_UNSAFE_HUGE_STRING, YSI_UNSAFE_HUGE_LENGTH, query, ___(3));
-	mysql_pquery(handle, YSI_UNSAFE_HUGE_STRING, "Indirect_FromCallback", "ii", _:cb, true);
+	new ret = mysql_format(handle, YSI_UNSAFE_HUGE_STRING, YSI_UNSAFE_HUGE_LENGTH, query, ___(3));
+	if(ret)
```

Comment:
Could you use `if (ret)` instead, just to be consistent with the rest of YSI?  Thanks.
```suggestion
	if (ret)
```